### PR TITLE
Remove indy-crypto

### PR DESCRIPTION
**Pull request by brentzundel**
This PR removes dependence on indy-crypto in code, comments, and documentation. It moves dependence to Ursa.

### CODE REVIEW

**Comment by brentzundel** [comment]
CI test this please

File: indy_common/config.py
Comment (skhoroshavin) [inline indy_common/config.py:100]: Should we also hold Ursa?

File: pool_automation/roles/indy_node/defaults/main.yml
Comment (Toktar) [inline pool_automation/roles/indy_node/defaults/main.yml:9]: Do we need adding Ursa there? I'm not good at pool_automation and not sure.

File: ci/ubuntu.dockerfile
Comment (brentzundel) [reply ci/ubuntu.dockerfile:10]: I can add that

File: dev-setup/osx/setup-dev-dependencies.sh
Comment (brentzundel) [reply dev-setup/osx/setup-dev-dependencies.sh:58]: There isn't a simple command line install for ursa on OSX, so I added instructions on installing from source in the setup-dev.md document

File: indy_common/config.py
Comment (brentzundel) [reply indy_common/config.py:100]: strictly speaking, ursa is a plenum dependency, not a node dependency, so if there is a way to do it in plenum, that's where we should do it.

File: ci/pipeline.groovy
Comment (brentzundel) [reply ci/pipeline.groovy:69]: I also do not know.

**Comment by brentzundel** [comment]
CI test this please

**Comment by brentzundel** [comment]
CI test this please

File: ci/pipeline.groovy
Comment (brentzundel) [reply ci/pipeline.groovy:69]: Who could review that may have an answer?

File: indy_common/config.py
Comment (brentzundel) [reply indy_common/config.py:100]: All I know is I removed those two packages because they are not needed.
I did not add ursa here because I was trusting that a test would fail if it was necessary to do so. Is that faulty logic?

File: pool_automation/roles/indy_node/defaults/main.yml
Comment (brentzundel) [reply pool_automation/roles/indy_node/defaults/main.yml:9]: Who would have the expertise to review this? I do not.

**Review by WadeBarnes — APPROVED** [review]
Reviewed CI updates, they look fine.

**Comment by WadeBarnes** [comment]
CI test this please

**Comment by WadeBarnes** [comment]
@skhoroshavin, I think this is ready to go once we have your approval.

**Review by skhoroshavin — APPROVED** [review]
I'm fine with merging this, but there is still a risk that CD pipeline might fail. If we can cope with it go ahead and merge.

**Comment by skhoroshavin** [comment]
@WadeBarnes It looks like CD pipeline failed, which means that as of now no release is possible until it is fixed.
https://build.sovrin.org/job/indy-node/job/indy-node-cd/job/master/

**Comment by WadeBarnes** [comment]
@skhoroshavin, I'll have a look.

**Pull request merged**
